keyboards/ploopyco/ploopyco.c

```diff
@@ -175,7 +175,7 @@ bool process_record_kb(uint16_t keycode, keyrecord_t* record) {
     }
 
     if (keycode == MOMENTARY_DRAG_SCROLL) {
-        is_drag_scroll = record->event.pressed;
+        is_drag_scroll ^= 1
```

Comment:
This was a subtle thing I wanted to change.
Behavior before this commit:
1. Press TOGGLE_DRAG_SCROLL and enter drag scroll mode.
2. Press and hold MOMENTARY_DRAG_SCROLL leaves drag scroll mode and enters mouse mode.
3. Release MOMENTARY_DRAG_SCROLL does nothing, the device stays in mouse mode instead of going back into drag scroll.

Behavior after this commit:
1. Press TOGGLE_DRAG_SCROLL and enter drag scroll mode.
2. Press and hold MOMENTARY_DRAG_SCROLL temporarily leaves drag scroll mode and enters mouse mode.
3. Release MOMENTARY_DRAG_SCROLL returns device to drag scroll mode
4. Press TOGGLE_DRAG_SCROLL to leave drag scroll mode.